components/api_server/src/routes/metric.py

```diff
@@ -4,22 +4,20 @@
 from typing import Any, cast
 
 import bottle
+from model.actions import copy_metric, move_item
+from model.defaults import default_metric_attributes
 from pymongo.database import Database
 
+from database.reports import insert_new_report, latest_report_for_uuids, latest_reports
 from shared.database.measurements import insert_new_measurement, latest_measurement, latest_successful_measurement
 from shared.model.metric import Metric
 from shared.utils.type import MetricId, SubjectId, Value
 from shared_data_model import DATA_MODEL
-
-from database.reports import insert_new_report, latest_report_for_uuids, latest_reports
-from model.actions import copy_metric, move_item
-from model.defaults import default_metric_attributes
 from utils.functions import sanitize_html, uuid
-
```

Comment:
Why are the imports reorganised? And why didn't ruff complain?

Reply:
Oh, that's weird! Probably incorrect IDE autoformatting

Reply:
Just checked, ruff doesn't raise any objections in either way the imports are organised. Perhaps it thinks they might be external package imports also?